examples/switch.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use druid::widget::{Column, Label, Padding, Row, Switch};
+use druid::{AppLauncher, Data, Lens, LensWrap, Widget, WindowDesc};
+
+#[derive(Clone, Data, Lens)]
+struct DemoState {
+    value: bool,
+}
+
+fn build_widget() -> impl Widget<DemoState> {
```

Comment:
this might be a bit too minimal of an example. potentially we could put switch inside another example, or expand this example to show off some other complementary aspects of druid. but I defer to @cmyr who has helped keep a handle on this directory.

Reply:
I find that when I'm _using_ sample code, I benefit when examples are more complex, use API in interesting ways, or otherwise show me things I might be less likely to figure out on my own.

So this is okay, but it would also be totally okay to add this to an existing example, to change behaviour in some way.

Reply:
I couldn't think of how existing examples could be extended in a useful way right now. But maybe there will be some better examples in the future that could replace this one.